index.js

```diff
@@ -118,6 +118,7 @@ function launchSpec(kernelSpec, spawnOptions) {
     const connectionFile = c.connectionFile;
     const config = c.config;
     const argv = kernelSpec.argv.map(x => x === '{connection_file}' ? connectionFile : x);
+    spawnObjects['env'] = Object.assign({}, process.env, kernelSpec.env);
```

Comment:
Oh wow! The test failed on my local and I just blindly committed. I think it's time for my second lunch....